crates/viewer/re_renderer/src/video/mod.rs

```diff
@@ -56,31 +48,35 @@ pub enum FrameDecodingResult {
     Error(DecodingError),
 }
 
+/// Identifier for an independent video decoding stream.
+///
+/// A single video may use several decoders at a time to simultaneously decode frames at different timestamps.
+/// The id does not need to be globally unique, just unique enough to distinguish streams of the same video.
+pub type VideoDecodingStreamId = u64;
```

Comment:
It would be nicer to make this a proper newtype

```suggestion
#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
pub struct VideoDecodingStreamId(pub u64);
```